plugins/woocommerce/src/Admin/API/OnboardingPlugins.php

```diff
@@ -0,0 +1,312 @@
+<?php
+/**
+ * REST API Onboarding Profile Controller
+ *
+ * Handles requests to /onboarding/profile
+ */
+
+namespace Automattic\WooCommerce\Admin\API;
+
+defined( 'ABSPATH' ) || exit;
+
+use ActionScheduler;
+use Automattic\WooCommerce\Admin\PluginsHelper;
+use Automattic\WooCommerce\Admin\PluginsInstallLoggers\AsynPluginsInstallLogger;
+use WC_REST_Data_Controller;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+
+/**
+ * Onboarding Plugins controller.
+ *
+ * @internal
+ * @extends WC_REST_Data_Controller
+ */
+class OnboardingPlugins extends WC_REST_Data_Controller {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc-admin';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'onboarding/plugins';
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/install-async',
+			array(
+				array(
+					'methods'             => 'POST',
+					'callback'            => array( $this, 'install_async' ),
+					'permission_callback' => array( $this, 'can_install_plugins' ),
+					'args'                => array(
+						'plugins' => array(
+							'description'       => 'A list of plugins to install',
+							'type'              => 'array',
+							'items'             => 'string',
+							'sanitize_callback' => function ( $value ) {
+								return array_map(
+									function ( $value ) {
+										return sanitize_text_field( $value );
+									},
+									$value
+								);
+							},
+							'required'          => true,
+						),
+					),
+				),
+				'schema' => array( $this, 'get_install_async_schema' ),
+			)
+		);
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/install-and-activate',
+			array(
+				array(
+					'methods'             => 'POST',
+					'callback'            => array( $this, 'install_and_activate' ),
+					'permission_callback' => array( $this, 'can_install_plugins' ),
```

Comment:
I think we'll need to also check `current_user_can( 'activate_plugins' )` for this particular endpoint

Reply:
Good suggestion! Updated in https://github.com/woocommerce/woocommerce/pull/38174/commits/1355fe3c963e54ce715091bed2457d2b45fe323b